README.md

```diff
@@ -291,9 +292,26 @@ npm start $service_id
 
 See [Ops Readme](ops/README.md).
 
-### Exporting a dataset
+### Dataset
 
-See [`export` script Readme](scripts/export/README.md).
+To generate a dataset:
+
+```
+npm run dataset:generate
```

Comment:
The `filename` option should be either documented or removed from implementation.